src-built-in/components/toolbarRefactored/app.tsx

```diff
@@ -0,0 +1,34 @@
+/*!
+* Copyright 2017 by ChartIQ, Inc.
+* All rights reserved.
+*/
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { Provider } from 'react-redux';
+
+import "../../../assets/css/font-finance.css";
+import "../../../assets/css/finsemble.css";
+import store from '../UIAPI/store';
+
+const ToolbarRefactored = () => {
+	return (<div className="finsemble-toolbar">
+		New Toolbar. Delete this line when the first component is added.
+		{/* Drag Handle */}
```

Comment:
All of this stuff (Drag Handle Through AutoArrange) comes in through props.children.

Reply:
got it